.github/workflows/CI.yml

```diff
@@ -239,24 +239,8 @@ jobs:
             exit 1
           fi
 
-      - name: build
-        run: pnpm -r build
-
-      - name: check-types
-        run: pnpm -r check-types
-
-      # To disable eslint-diff just comment the following two lines
-      - name: Run eslint-diff
-        run: pnpm -r run lint-diff
-
-      - name: Run eslint-diff-canary to make sure eslint-diff is functioning properly
-        run: pnpm -F commonwealth run lint-diff-canary
-
-      # fun the *old* linter last because I want to try to migrate away from
-      # this and JUST to eslint-diff but if this fails we still might be able
-      # to salvage the branch
-      - name: Run linter
-        run: pnpm lint-branch
+      - name: Lint and Check types
+        run: pnpm check-all
```

Comment:
I think this is going to mangle all the lines together.  When something fails we won't know which command caused the error.  

And if multiple commands generate errors they're going to be all intermingled together.